repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -1573,15 +1573,15 @@ private EntityMutationResponse deleteVertices(Collection<AtlasVertex> deletionCa
         RequestContext         req      = RequestContext.get();
 
         Collection<AtlasVertex> categories = new ArrayList<>();
-        Collection<AtlasVertex> other = new ArrayList<>();
+        Collection<AtlasVertex> others = new ArrayList<>();
```

Comment:
Why renamed this?

Reply:
since it hold collection of vertices and so renamed to plural keyword, to make it little easy for understanding.